index.html

```diff
@@ -49,43 +50,48 @@
             </div>
         </div>
         <div class="site-hero">
-        <div class="fling-minislide">
-            <img src="images/zdj1.jpg" alt="Slide 4" />
-            <img src="images/zdj3.jpg" alt="Slide 3" />
+            <div class="fling-minislide">
```

Comment:
rozumiem że ta karuzela jest ostylowana z height: 900px ze względu na wymiary tych zdjęć? tak czy siak troche dziwnie to wygląda jak się wchodzi na stronę i jest wielkie zdjęcie a pod nim biały pasek. Może lepiej dać height: 100vh; ? co myślisz? poza tym nie myśleliście żeby wrzucić jakiś napis na tą karuzelę czy coś?